sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -124,9 +124,14 @@ public void execute(SensorContext context) {
 
     Map<String, Integer> numberOLinesOfCode = new HashMap<>();
 
-    analyseFiles(context, phpAnalyzer, inputFiles, progressReport, numberOLinesOfCode);
-    if (inSonarQube(context)) {
-      processCoverage(numberOLinesOfCode, context);
+
+    try {
+      analyseFiles(context, phpAnalyzer, inputFiles, progressReport, numberOLinesOfCode);
+      if (inSonarQube(context)) {
+        processCoverage(numberOLinesOfCode, context);
+      }
+    } catch (CancellationException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)